src/github/pull_requests.graphql

```diff
@@ -1,19 +1,69 @@
-query PullRequests($owner: String!, $name:String!, $first:Int, $last:Int, $before:String, $after:String) {
-  repository(owner: $owner name: $name) {
-    pullRequests(first: $first last: $last before: $before after: $after) {
+query PullRequests(
+  $owner: String!
+  $name: String!
+  $first: Int
+  $last: Int
+  $before: String
+  $after: String
+) {
+  repository(owner: $owner, name: $name) {
+    pullRequests(first: $first, last: $last, before: $before, after: $after) {
       pageInfo {
-        hasNextPage,
-        endCursor,
+        hasNextPage
+        endCursor
       }
       nodes {
-        number,
-        title,
+        id
+        number
+        title
+        body
+        state
+        createdAt
+        updatedAt
+        closedAt
+        mergedAt
+        author {
+          __typename
+          ... on User {
+            login
+          }
+        }
+        additions
+        deletions
+        url
+        repository {
+          owner {
+            __typename
+            login
+          }
+          name
+        }
+        labels(last: 5) {
```

Comment:
The discussion from https://github.com/aicers/github-dashboard-server/pull/173#discussion_r2149462427 applies here as well. Please add a `TODO` comment for all items limited to a specific number.